src/NzbDrone.Core/DecisionEngine/Specifications/RawDiskSpecification.cs

```diff
@@ -13,7 +13,7 @@ public class RawDiskSpecification : IDecisionEngineSpecification
                                                     {
                                                         new Regex(@"(?:dis[ck])(?:[-_. ]\d+[-_. ])(?:(?:(?:480|720|1080|2160)[ip]|)[-_. ])?(?:Blu\-?ray)", RegexOptions.Compiled | RegexOptions.IgnoreCase),
                                                         new Regex(@"(?:(?:480|720|1080|2160)[ip]|)[-_. ](?:full)[-_. ](?:Blu\-?ray)", RegexOptions.Compiled | RegexOptions.IgnoreCase),
-                                                        new Regex(@"(?:\d?x?M?DVD-?[R59])", RegexOptions.Compiled | RegexOptions.IgnoreCase)
+                                                        new Regex(@"(?:\d?x?M?DVD-?[R59])\b", RegexOptions.Compiled | RegexOptions.IgnoreCase)
```

Comment:
```suggestion
                                                        new Regex(@"(?:\d?x?M?DVD-?[R59])[ ._]", RegexOptions.Compiled | RegexOptions.IgnoreCase)
```

So this doesn't fail if underscores are used as a separator